go/tools/gazelle/resolve/resolve.go

```diff
@@ -77,6 +77,12 @@ func NewLabelResolver(c *config.Config) LabelResolver {
 	switch c.DepMode {
 	case config.ExternalMode:
 		e = newExternalResolver()
+	case config.KnownMode:
+		er := newExternalResolver()
```

Comment:
If I'm adding KnownImports to config.Config{...} should I move the default known imports from resolve_external.go to config.go?